docs/dedicated-data-endpoints/dedicated-data-endpoints.md

```diff
@@ -0,0 +1,115 @@
+# Azure Container Registry: Dedicated Data Endpoints – Mitigating Data Exfiltration
+
+Azure Container Registry announces dedicated data-endpoints, enabling client firewall rules on registries, minimizing data exfiltration concerns.
```

Comment:
"enabling client firewall rules on registries" sounds like the registry is supporting "client firewall  rules". Shouldn't this be something like enabling customers to create more scoped client firewall rules in their environments?